tests/perf/README.md

```diff
@@ -0,0 +1,31 @@
+# Performance Testing
+
+## Dependencies
+To use bcc and eBPF, you will need to [install bcc](https://github.com/iovisor/bcc/blob/master/INSTALL.md). If using Ubuntu, we recommend following the instructions to [build from source](https://github.com/iovisor/bcc/blob/master/INSTALL.md#ubuntu---source).
+
+If you want to run the scripts in a Python virtual environment, you can run:
+```
+python3 -m venv env
+source env/bin/activate
+python3 -m pip install -U matplotlib
+pip install pandas
+pip install hdrhistogram
+pip install tomli-w
+source env/bin/activate
+```
+
+## Number of Subscriptions vs. Function Latency
+`generate_ip_subs.py` shards the IPv4 address space into `n` subnets to generate `n` Retina subscriptions, where `n` is passed in by the user. The subscriptions are written to `spec.toml`.
```

Comment:
Maybe clarify that this is a sample / basic application and more can easily be added. The main goal of your project was to set up the infrastructure.